src/main/java/gov/usgs/owi/nldi/springinit/WebSecurityConfig.java

```diff
@@ -0,0 +1,20 @@
+package gov.usgs.owi.nldi.springinit;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+
```

Comment:
Unfortunately, spring security is needed for the csrf functionality, forcing us to permit all access to all endpoints.